lib/soapy_cake/admin_batched.rb

```diff
@@ -53,7 +54,7 @@ def next_batch
     def method_missing(name, method_opts = {}, limit = nil)
       fail Error, "Invalid method #{name}" unless ALLOWED_METHODS.include?(name)
 
-      BatchedRequest.new(admin, name, method_opts, limit).to_enum
+      BatchedRequest.new(admin, name, method_opts.merge(opts.slice(:xml_response)), limit).to_enum
```

Comment:
Don't you rather want a new parameter for that? Does it really make sense to add this flag to the cake API opts?